packages/thirdweb/src/extensions/vote/write/executeProposal.ts

```diff
@@ -0,0 +1,63 @@
+import type { BaseTransactionOptions } from "../../../transaction/types.js";
+import {
+  type ExecuteParams,
+  execute,
+} from "../__generated__/Vote/write/execute.js";
+import { proposalExists } from "../read/proposalExists.js";
+
+/**
+ * Execute a Proposal
+ * @extension VOTE
+ * @returns a prepared transaction for the `execute` method
+ *
+ * @example
+ * ```ts
+ * import { executeProposal } from "thirdweb/extensions/vote";
+ *
+ * const transaction = executeProposal({ contract, proposalId });
+ * const tx = await sendTransaction({ transaction, account });
+ * ```
+ */
+export function executeProposal(
```

Comment:
would be great to add a test for these new extensions, but can be done separately